src/AbsMSeries.jl

```diff
@@ -154,53 +155,53 @@ end
 RandomExtensions.maketype(S::MSeriesRing, _, _) = elem_type(S)
 
 function RandomExtensions.make(S::MSeriesRing,
-                                      term_range::AbstractUnitRange{Int}, vs...)
-   R = base_ring(S)
-   if length(vs) == 1 && elem_type(R) == Random.gentype(vs[1])
-      Make(S, term_range, vs[1])
-   else
-      Make(S, term_range, make(R, vs...))
-   end
+    term_range::AbstractUnitRange{Int}, vs...)
```

Comment:
here it doesn't do it, but it does the opposite?

Reply:
I don't know what the internal logic of the Julia-Vim plugin does. Maybe it reacts to the number of open brackets?